### ISSUE TITLE
Forge mod compatibility - Controllable 

### ISSUE THREAD

**Issue by shoya4000** [jrbudda/Vivecraft_115#25]
When running Vivecraft 1.15.2 v3 with Forge 31.2.31 and Controllable 0.8.0, it appears to break.

When using Forge 31.2.31 and Controllable 0.8.0, using an Xbox One controller works as expected.

When using an Xbox One controller and also using Vivecraft, the right stick will not initiate turning as expected.
Likewise, the movement directions on the left stick do not line up with the direction faced
- especially as right stick will not cause player to turn, user would expect pressing forward on the left stick to move in direction user is facing, however it seems to be locked to some other direction.

Attempting to set up use of Xbox One controller for seated play.
Using Oculus Rift S

**Comment by Techjar** [comment]
Yeah this has been brought up multiple times. Controller mods won't work with Vivecraft because we (necessarily) have custom code for handling mouselook in seated mode. You should be able to fix the movement direction by messing with the setting for that.
